reaktive-test/src/commonMain/kotlin/com/badoo/reaktive/test/base/TestObserverExt.kt

```diff
@@ -0,0 +1,31 @@
+package com.badoo.reaktive.test.base
+
+import com.badoo.reaktive.test.utils.assertEquals
+import com.badoo.reaktive.test.utils.assertFalse
+import com.badoo.reaktive.test.utils.assertTrue
+
+val TestObserver.isError: Boolean get() = error != null
+
+fun TestObserver.assertError() {
```

Comment:
If we have access to `TestObserver` classes, why do we need extension functions? Put them inside corresponding classes to make them more visible and accessible?

Reply:
Discussed in person